eth/eventdatahandler/task_executor_test.go

```diff
@@ -24,8 +25,8 @@ const rawValidatorAdded = `{
 		"transactionHash": "0x921a3f836fb873a40aa4f83097e52b69225334c49674dc262b2bb90d27e3a801"
 	  }`
 
-func TestExecuteTask(t *testing.T) {
-	logger := zaptest.NewLogger(t)
+func TestExecuteStartValidatorTask(t *testing.T) {
```

Comment:
What do you think about having one test for all tasks with sub-tests for each of them?

Reply:
I prefer separated tests as it follows better KISS pattern, but yeah lets keep everything in the same style.

Reply:
alright, let's keep them separated then